### DISCUSSION TITLE
ΑΝΑΒΟΛΗ ΜΑΘΗΜΑΤΟΣ IV ΔΕΥΤΕΡΑ 22/11

### DISCUSSION THREAD

**Discussion by vournam** [upatras-hci/iv#16] [Q&A]
Καλησπέρα σας.
Θα μπορούσαμε να αφήσουμε αυτη την εβδομάδα κενή και να συναντηθούμε για απορίες την επόμενη εβδομάδα;(ωστε να έχουμε ασχοληθεί αρκετά με τα παραδοτέα.)

Με εκτίμηση,
Βουρνά Μαρία-Μελίνα
Φιλιοπούλου Διονυσία

**Comment by epidrome** [comment]
ναι, όποτε θέλετε και έχει γίνει πρόοδος μπορούμε να κάνουμε μια συνάντηση
